src/Document/Metadata/MetadataInterface.php

```diff
@@ -23,7 +23,7 @@ public function getFilterableAttributes(): array;
     /**
      * Returns the names of the filterable attributes
      *
-     * @return list<string>
+     * @return array<string>
```

Comment:
Why do you change it to something less specific?

Reply:
🤷